### ISSUE TITLE
Run without admin privileges

### ISSUE THREAD

**Issue by bitching** [FlowerWrong/tun2socks#37]
using netsh to set static ip requires privilege elevation, but this can be avoid by using dhcp.
this helps those one who doesn't want to modify route tables.

See:
https://github.com/yinghuocho/gotun2socks/blob/master/tun/tun_windows.go#L161

**Comment by FlowerWrong** [comment]
Good idea. Can you mention a PR? And add some note about administrator privilege on windows [here](https://github.com/FlowerWrong/tun2socks/blob/master/config.example.ini#L37-L48)

**Comment by bitching** [comment]
I can barely read some codes(not much), and not able to produce them, which is embarrassing.
